examples/openshift/README.md

```diff
@@ -1,10 +1,8 @@
-OpenShift example
-=================
+# OpenShift example
```

Comment:
:heart: I saw this before and was thinking that prettier would probably manage to fix this :)